src/Components/SingleReview.jsx

```diff
@@ -1,7 +1,35 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchSingleReview } from "../api";
+
 function SingleReview() {
-    return (
-      <p>This will be a single review.</p>
-    );
-  }
-  
-  export default SingleReview;
\ No newline at end of file
+  const [currentReview, setCurrentReview] = useState({});
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    fetchSingleReview(review_id)
+      .then((review) => setCurrentReview(review))
```

Comment:
I'd consider a loading pattern here, when throttling the connection the display doesn't tell the user what is going on.